boggle-sveltexpress/svelte/src/App.svelte

```diff
@@ -2,6 +2,7 @@
   import { Router, Route } from "svelte-routing";
   import { Room, NotFound, Home } from "@routes";
   import GboggleLogo from "@components/GBoggleLogo.svelte";
+import RoomBrowser from "./routes/RoomBrowser.svelte";
```

Comment:
Add RoomBrowser to ./routes/index.ts and import it with the other routes (`import { Room, NotFound, Home, RoomBrowser } from "@routes";`)

Reply:
It's already added to the index.ts, so just import it from "@routes"